drivers/ksfilter/ks/connectivity.c

```diff
@@ -284,9 +284,13 @@ KspReadMediaCategory(
 
     /* query the name size */
     Status = ZwQueryValueKey(hKey, &Name, KeyValuePartialInformation, NULL, 0, &Size);
+
+    DPRINT("ZwQueryValueKey() status 0x%08lx %wZ\n", Status, &Name);
```

Comment:
As much as you say the `DPRINT` calls helped you tracking down the problem you were investigating, would it make sense to log in the NTSTATUS code even on a success case? Why?

Reply:
Why `Name`, which is already known? And not `Size`, which is queried?

> status is always successful

That feels rather odd: `STATUS_BUFFER_TOO_SMALL` seems expected in this pattern...